css/estilos-index.css

```diff
@@ -0,0 +1,196 @@
+@import url(flexboxgrid.min.css);
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@400;600;700&display=swap');
+
+*{
+    margin: 0;
+    padding: 0;
+}
+#enlace-logo{
+    width: 0;
+    float: left;
+    padding: 0;
+    margin: 0 0 0 0;
+}
+#logo-cine{
+    width: 120px;
+    float: center;
+    padding: 0;
+    margin: 0 0 0 0;
+}
+#nombre-cine{
+    color: #3C64B1;
+    width: 15%;
+    float: left;
+    font-size: 24px;
+    margin: 30px 0 0 75px;
+    font-family: 'Lato', sans-serif;
+}
+nav{
+    display: flex;
+    width: 105%;
+    left: 0;
+    top: 0;
+    position:fixed;
+    flex-direction:row;
+}
+nav ul {
+    background-color: hsl(219, 49%, 46%, .3);
+    list-style: none;
+    justify-content: space-around;
+    padding: 0;
+    margin: 0;
+}
+nav li a {
+    color: #373F41;
+    display: inline-block;
+    align-items: flex-end;
+    text-decoration: none;
+    padding: .5em;
+    margin: 30px 0px 5px 0px;
+    font-size: 14px;
+    font-family: 'Mulish', sans-serif;
+}
+nav li a:hover {
+    background: tomato;
+    color: white;
+    text-decoration: underline;
+  }
+
+#acceso{
+    background-color:  #3C64B1;
+    color: #fff;
+    border: none;
+    letter-spacing: 1px;
+    text-transform: unset;
+    text-align: center;
+    cursor: pointer;
+    padding: 1em;
+    margin: 30px 30px 0 0;
+    font-size: 14px; 
+}
+
+#acceso:hover{
+    background-color: #ffba08;
+    text-decoration: underline;  
+}
+/*SECCION 1*/
+.seccion-uno{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+    margin-top: 150px;
+    margin-bottom: 85px;
+    margin-right: 5px;
+    margin-left: 5px;
+    align-items: center;
+}
+#tituloseccion1{
+    padding: 5px;
+    color:  #373F41;
+    font-size: 44px;
+    margin-top: 5px;
+    text-align: left;
+    font-family: 'Mulish', sans-serif;
+}
+#seccion1{
+    color: #737B7D;
+    margin: 5px 5px 5px 5px;

```

Comment:
These values could be reduced to `5px` as the next line. One value will be applied to the 4 sides